influxql/parser_test.go

```diff
@@ -293,6 +293,19 @@ func TestParser_ParseStatement(t *testing.T) {
 			},
 		},
 
+		// SHOW TAG VALUES WITH KEY = "..."
+		{
+			s: `SHOW TAG VALUES WITH KEY = "host" WHERE region = 'uswest'`,
+			stmt: &influxql.ShowTagValuesStatement{
+				TagKeys: []string{`"host"`},
```

Comment:
why is the parser yielding the quotes? I would expect the parser to take care of clearing those out. Anywhere an identifier is yielded it should be cleared of quotes.